src/main/scala/org/ua/ilm/Entity/JsonPars.scala

```diff
@@ -0,0 +1,23 @@
+package org.ua.ilm.Entity
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.scala.experimental.ScalaObjectMapper
+
+object JsonPars {
+  val mapper = new ObjectMapper() with ScalaObjectMapper
+
+  def parsJson(json: String, key: String): Unit = {
+    val jsonNode = mapper.readTree(json)
+    println(jsonNode.findValue(key))
```

Comment:
*parse. Also, there is no actual need in this debug code. YoutubeChannelParser class with "parse" method will be enough.